nss-day-cohort-46/rare-rest-rare-api-cool-cake
# Pp edit comment

Description:

Backend for editing a comment is finished

Testing Instructions:

Postman URL:
Change the id of the URL to a comment that you have in the database, if you don't have id = 2.
```
http://localhost:8000/comments/2
```

Postman Object:
{
    "postId": 1,
    "content": "edited new comment",
    "createdOn": "2016-05-18T15:37:36.993048Z"
}

Make a PUT request and check the database to see if the post was changed.
